apps/daimo-mobile/src/view/screen/send/SendNoteButton.tsx

```diff
@@ -68,26 +71,28 @@ function SendNoteButtonInner({
       timestamp: Date.now() / 1e3,
       nonceMetadata: nonce.metadata.toHex(),
     },
-    accountTransform: (account, pendingOp) => ({
-      ...account,
-      pendingNotes: [
-        ...account.pendingNotes,
+    accountTransform: (account, pendingOp) => {
+      const newAccount = transferAccountTransform([
         {
-          type: "note",
-          ephemeralOwner,
-          ephemeralPrivateKey: ephemeralPrivKey,
-          previewDollars: `${dollars}`,
-          previewSender: account.name,
-          opHash: pendingOp.opHash,
-        },
-      ],
-    }),
-    namedAccounts: [
-      {
-        addr: daimoEphemeralNotesAddress,
-        label: AddrLabel.PaymentLink,
-      } as EAccount,
-    ],
+          addr: daimoEphemeralNotesAddress,
+          label: AddrLabel.PaymentLink,
+        } as EAccount,
+      ])(account, pendingOp);
```

Comment:
double function call is a bit hard to read

the first part is constant, right?

could do
```ts
const addPendingNote = transferAccountTransform(...);
```
at module scope, then here just

```ts
const newAccount = addPendingNote(account, pendingOp);
return {...};
```

Reply:
done